src/Components/Form/Form.tsx

```diff
@@ -10,8 +10,13 @@ import { StyleSheet, css } from "aphrodite";
 import { palette, typography } from "styles/index";
 import { useTranslation } from "react-i18next";
 import { Icons } from "constants/enums/Icons";
-import { Field, FormikProvider, useFormik } from "formik";
+import { useFormik } from "formik";
 import * as Yup from "yup";
+import { useDispatch } from "react-redux";
+import { addTodo } from "store/todo/todoActions";
+import { Message } from "constants/types/Message";
+import { useAlert } from "react-alert";
+import { Category, Priority, TodoType } from "constants/types/TodoTypes";
```

Comment:
There are more and more imports, so you should create additional index.ts in src/constants